app/models/person.rb

```diff
@@ -1,12 +1,21 @@
 # frozen_string_literal: true
 
 class Person < VersionedModel
-  has_many :profiles, dependent: :destroy
+  IDENTIFIER_TYPES = %i[
+    police_national_computer criminal_records_office prison_number
+  ].freeze
 
+  has_many :profiles, dependent: :destroy
   has_many :moves, through: :profiles
 
+  belongs_to :ethnicity, optional: true
+  belongs_to :gender, optional: true
+
   has_one_attached :image
 
+  validates :last_name, presence: true
```

Comment:
before we run the backfilling task will this not cause existing records to be invalid?

Reply:
Yes it will. I'll remove it, but leave the fixes for the test failures it caused by adding it.